backend/setup/setup.go

```diff
@@ -70,11 +70,24 @@ func LoadEconomicsConfig() (*EconomicConfig, error) {
 // EconConfigLoader allows functions to use this type as a parameter to load an EconomicConfig Dependency
 type EconConfigLoader func() *EconomicConfig
 
+// MarketCreationLoader loads just the market creation component of the setup
+type MarketCreationLoader interface {
+	LoadMarketCreation() MarketCreation
+}
+
```

Comment:
Focusing on this part of the code, I was struggling to understand why, "market creation," is a part of the setup. However after reviewing the entire codebase, I see that it's simply a way of creating brevity, e.g. so we don't have to do: `this.thing.then.this.thing` ... Part of my confusion I think was, the act of creating a market is an action that you can do within our application, e.g., "create market," whereas, this is more like, a state, with parameters used in the process of creating a market.

Would `MarketCreationCostsLoader` or, `MarketCreationIncentivesLoader` be any clearer from your point of view?

I also understand the desire to keep variable names shorter, but at least at work, I feel like there are a lot of super short names which are shared between different abstractions. E.g., "data," is used, rather than, "runInputData" is the convention, however you might see, "data" used in different ways across different parts of our software.

Might be an aesthetic preference, might be a way of making things more clear. What are your thoughts?

Reply:
I think if we want to rename this interface we should consider renaming the [MarketCreation type](https://github.com/openpredictionmarkets/socialpredict/pull/327/files#diff-2968ab25ba40f08fb5caa3e34536c3c87bcc2862565bb58599a4b4e1b4d4da99L14-L19) as well.
This could be a `setup.Market`, `setup.InitialMarket` (and then rename the fields within to be less redundant). We're going to be using the setup consistently throughout this code base, so it might be worth having a video call / brainstorming session to think about the ergonomics of these names.